services/ui-src/src/forms/mcpar/cpc/cpc.schema.ts

```diff
@@ -1,3 +1,22 @@
 import { object } from "yup";
+import {
+  checkbox,
+  date,
+  nested,
+  number,
+  radio,
+  text,
+} from "utils/forms/schemas";
 
-export default object({});
+export default object({
```

Comment:
Change the keys here to match changes in other file: 

`cpc-1-b` → `cpc-1-sub`
`cpc-4` → `cpc-4a`
`cpc-4-o5-text `→ `cpc-4a-o5-text`
`cpc-5` → `cpc-4b`
`cpc-6` → `cpc-5`
`cpc-7` → `cpc-6`